### PR TITLE
[EOC-311] Cohort do not delete flag as deleted

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: server/common/utils/helpers.test.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ describe('function responseWithCohorts', () => {
     );
   });
 
-  const notExpected = ['favIds', 'ownerIds', 'memberIds'];
+  const notExpected = ['favIds', 'isDeleted', 'ownerIds', 'memberIds'];
```
Comment (Ciunkos) [inline server/common/utils/helpers.test.js]: Consider sorting this.

File: server/common/utils/helpers.test.js [hunk 2]
```diff
@@ -174,7 +174,7 @@ describe('function responseWithCohort', () => {
...
```
Comment (Ciunkos) [inline server/common/utils/helpers.test.js]: This as well.

File: server/controllers/cohort.js [hunk 3]
```diff
@@ -218,11 +219,21 @@ const deleteCohortById = (req, resp) => {
       if (lists) {
         const listsIds = lists.map(lists => lists._id);
 
-        return Comment.deleteMany({ listId: { $in: listsIds } });
+        return Comment.updateMany(
+          { listId: { $in: listsIds } },
```
Comment (Ciunkos) [inline server/controllers/cohort.js]: Shouldn't the listIds be sanitized as well?

Comment (adamklepacz) [reply server/controllers/cohort.js]: Those listsIds are coming from the database, not from the request of the client, so in my opinion, it doesn't have to be sanitized. Those ids are generated by MongoDB itself, so there will be no query injections.

Comment (Ciunkos) [reply server/controllers/cohort.js]: Ok

**Pull request merged**